web/templates/recipient_row.hbs

```diff
@@ -89,9 +89,9 @@
         <span class="private_message_header_name">
             {{#tr}}You and <z-user-names></z-user-names>
             {{#*inline "z-user-names"}}
-                {{#each recipient_users}}
-                    {{full_name}}{{#if should_add_guest_user_indicator}}&nbsp;<i>({{t 'guest'}})</i>{{/if}}{{#unless @last}},{{/unless}}
-                {{/each}}
+                {{#each recipient_users~}}
+                    {{full_name}}{{#if should_add_guest_user_indicator}}&nbsp;<i>({{t 'guest'}})</i>{{/if}}{{#unless @last}},&nbsp;{{/unless}}
```

Comment:
```suggestion
                    {{full_name}}{{#if should_add_guest_user_indicator}}&nbsp;<i>({{t 'guest'}})</i>{{/if}}{{#unless @last}}, {{/unless}}
```

[Discussion](https://chat.zulip.org/#narrow/stream/43-automated-testing/topic/message-basics.2Etest.2Ets.20failure/near/1832448) (by the way, when starting a discussion about a PR on chat.zulip.org, remember to link back from the PR to the discussion, so that someone reviewing the PR has all the context).